.github/workflows/build-kata-static-tarball-amd64.yaml

```diff
@@ -21,6 +21,8 @@ jobs:
           - kernel
           - kernel-dragonball-experimental
           - kernel-tdx-experimental
+          - kernel-gpu-snp
+          - kernel-gpu-tdx
```

Comment:
I like it.

There are two things that make me a little bit concerned, though.
1) We don't ship a SNP kernel yet
2) I'd prefer renaming `kernel-gpu-tdx` `kernel-gpu-tdx-experimental`, as the TDX kernel is not yet upstreamed (and the same applies for SNP, I think).
3) I'd also love to see the GPU being provided for the "vanilla" kernel, but this one can be a follow-up PR.

Reply:
We ship a SNP guest-kernel the 5.19.2 its in the version.yaml. Vanilla yep make sense.